scripts/app

```diff
@@ -106,7 +106,7 @@ compose() {
   local env_file="${UMBREL_ROOT}/.env"
   local app_base_compose_file="${UMBREL_ROOT}/apps/docker-compose.common.yml"
   local app_compose_file="${app_dir}/docker-compose.yml"
-  local app_domain="$(hostname -s 2>/dev/null || echo "umbrel").local"
+  local app_domain="$(cat /proc/sys/kernel/hostname -s 2>/dev/null || echo "umbrel").local"
```

Comment:
It looks like you've left the `-s` from the hostname command

Reply:
Good catch. Thank you.